### PR TITLE
Updated [PT]/lessons/Specific/Plug.md ✍️💅

### PR DESCRIPTION
**Pull request by rodrigopinto**
This PR is for updating the Plug section for Portuguese according to the most recent version in English.

I've committed the updates for each section and there is the last one that does a general review on typos. After the PR review, I'll squash it to a single commit, to make easier the merge.

### CODE REVIEW

**Comment by doomspork** [comment]
This is great, thank you @rodrigopinto!  We'll get a Portuguese translator to take a peek at this 😁

**Comment by pragmaticivan** [comment]
Thanks @rodrigopinto I'm currently reviewing that! 👍

File: pt/lessons/specifics/plug.md [hunk 1]
```diff
@@ -1,64 +1,183 @@
 ---
-version: 0.9.0
+version: 1.1.1
 title: Plug
 ---
 
-Se você estiver familiarizado com Ruby, você pode pensar sobre Plug como o Rack com uma pitada de Sinatra, ele fornece uma especificação para componentes de aplicação web e adaptadores para servidores web. Mesmo não fazendo parte do núcleo de Elixir, Plug é um projeto oficial de Elixir.
+Se você estiver familiarizado com Ruby, você pode pensar sobre Plug como o Rack com uma pitada de Sinatra. Ele fornece uma especificação para componentes de aplicação web e adaptadores para servidores web. Mesmo não fazendo parte do núcleo de Elixir, Plug é um projeto oficial de Elixir.
+
+Nós iremos começar criando uma mini aplicação web baseada no Plug. Depois, iremos aprender sobre as routas do Plug e como adicionar um Plug à uma aplicação web existente.
```
Comment (pragmaticivan) [inline pt/lessons/specifics/plug.md]: maybe instead of `routas` use `rotas`?

Comment (rodrigopinto) [reply pt/lessons/specifics/plug.md]: 🙈 Ops! For sure. There is always some typo, it does not matter how many times we review. Thanks for pointing.

**Comment by rodrigopinto** [comment]
@pragmaticivan Fixed the typo, should I squash already? Apart for this PR, the specs aren't passing atm, is it fine?

File: pt/lessons/specifics/plug.md (hunk 1 above)
Comment (pragmaticivan) [reply pt/lessons/specifics/plug.md]: I feel you, bro! haha Happens to me all the time haha

**Comment by pragmaticivan** [comment]
Hum.. good question, I have no rights to merge, I would just approve. Maybe just ensure that this PR is rebased from master and we're good to go. The CI seems to be broken for other reasons.

**Comment by doomspork** [comment]
Thanks @rodrigopinto and @pragmaticivan!  I will look into the failing specs 👌

**Pull request merged**